src/simtools/schemas/model_parameters/dsum_ignore_below.schema.yml

```diff
@@ -24,7 +24,6 @@ instrument:
   type:
     - MSTN
     - MSTS
-    - SSTS
```

Comment:
Why did you remove only SSTs if it is relevant only for FlashCams?

Reply:
Same, see above.